guvnor-workingset/guvnor-workingset-client/src/main/java/org/guvnor/common/services/workingset/client/WorkingSetManager.java

```diff
@@ -58,7 +58,7 @@
     public void onWorkingSetApplied(@Observes final OnWorkingSetApplied event) {
 //        final Pair<Path, WorkingSetSettings> projectReference = getProjectConfig( event.getResource() );
 //        if ( projectReference != null && projectReference.getK2() == null ) {
-//            projectService.call( new RemoteCallback<WorkingSetSettings>() {
+//            moduleService.call( new RemoteCallback<WorkingSetSettings>() {
```

Comment:
Tried to remove this class, but Guided Rule Editor depends on this so afraid to remove it.